packages/compartment-mapper/src/node-modules.js

```diff
@@ -252,24 +252,49 @@ const graphPackage = async (
   graph[packageLocation] = /** @type {Node} */ (result);
 
   /** @type {Record<string, string>} */
-  const dependencies = {};
+  const dependenciesObj = {};
```

Comment:
Please consider a meaningful distinction in the name.
```suggestion
  const dependencyLocations = {};
```